src/main/java/org/junitpioneer/jupiter/json/JacksonJsonConverter.java

```diff
@@ -13,29 +13,69 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.UncheckedIOException;
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
 
 import com.fasterxml.jackson.core.JsonParser;
 import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.Module;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import org.junitpioneer.internal.PropertyUtils;
+
 /**
  * A {@link JsonConverter} using Jackson 2 {@link ObjectMapper} to perform the conversion
  */
 class JacksonJsonConverter implements JsonConverter {
 
+	private static final String PROPERTY_PREFIX = "junitpioneer.jackson.modules";
```

Comment:
```suggestion
	private static final String PROPERTY_PREFIX = "junit.pioneer.jackson.modules";
```

Reply:
this needs to stay consistent with the other property that's parsed above (`property-utils`). either use a `.` or a `-`